### ISSUE TITLE
make the about/more link stand out on the login page

### ISSUE THREAD

**Issue by jennifermf** [tootsuite/mastodon#7158]
I keep pointing people with questions to the about/more page, and they keep saying they didn't realize the page was there. Can we please make this link MUCH more obvious for new and old users alike?

I think you should leave the default "the rules of the service" text where it is, and add a second link/button under the signup, or in the short blurb about the instance. (such as "to learn more about this instance....")

we're putting a ton of great info in our about/more pages... let's make it stand out and easier to find.

![screen shot 2018-04-16 at 12 33 07 ptm](https://user-images.githubusercontent.com/26920220/38830898-8eae6f06-4172-11e8-967e-f284a6e2ee0f.png)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by ruthtillman** [comment]
Yes, I've been working on the community guidelines and overall intro stuff for an instance and we really want this up front as much as possible.

**Comment by dolljoints** [comment]
also i recently got a friend to sign up, and they didn't know how to follow someone from another instance. they went to the faq page, but they found it took them 45 minutes to find the info. i went and had a look and the relevant things (how to actually social) are almost at the bottom, below a big glossary. 

if i need to make this a new github issue i can, it just seemed relevant because both are part of the first-time experience being frustrating.
